source/elements/oneMKL/source/domains/sparse_linear_algebra.inc.rst

```diff
@@ -9,29 +9,21 @@ Sparse Linear Algebra
 
 .. container::
 
+   The oneAPI Math Kernel Library provides a Data Parallel C++ interface to some
+   of the Sparse Linear Algebra routines.
```

Comment:
I have reworded this in https://github.com/oneapi-src/oneAPI-spec/pull/522/commits/13825be51f602bbe1ca22f11a0844fe4d5ee2106. I prefer to avoid using "/" which doesn't sound proper English to me.